webdriver/tests/bidi/browsing_context/user_prompt_opened/user_prompt_opened.py

```diff
@@ -56,6 +56,28 @@ async def test_prompt_type(
     }
 
 
+async def test_prompt_default_value(bidi_session, subscribe_events, inline, new_tab,
+                           wait_for_event, prompt_type):
```

Comment:
The indentation is off. Maybe you can fix it? 

Also `prompt_type` is from a parameterization of another test and not used here. It should basically fail that test. Interesting that it doesn't.

```suggestion
async def test_prompt_default_value(
    bidi_session, inline, new_tab, subscribe_events, wait_for_event
):
```